doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+2. Once you know the source is running, use the [`mz_source_statistics`](/sql/system-catalog/mz_internal/#mz_source_statistics) table to check the status of the initial snapshot:
+
+    ```sql
+    SELECT bool_and(snapshot_committed)
+    FROM mz_internal.mz_source_statistics
+    WHERE id = '<source_ID>';
+    ```
+    <p></p>
+
+    ```noftm
+     snapshot_committed
+    --------------------
+     t
+    (1 row)
+    ```
+
+    Once `snapshot_commited` is `t`, move on to the next step. Snapshotting can take between a few minutes to several hours, depending on the size of your dataset and the size of the cluster replica you chose for your `ingest_rds_postgres` cluster.
+
+## Step 7. Right-size the cluster
+
+After the snapshotting phase, Materialize starts ingesting change events from the PostgreSQL replication stream. For this work, Materialize generally performs well with an `xsmall` replica, so in this step, you'll resize the replica in your source cluster.
+
+1. Still in the `psql` shell connected to Materialize, remove the `medium` replica from the source cluster:
+
+    ```sql
+    DROP CLUSTER REPLICA ingest_rds_postgres.r1;
+    ```
+
+1. Add an `xsmall` replica to the source cluster:
+
+    ```sql
+    CREATE CLUSTER REPLICA ingest_rds_postgres.r1 SIZE 'xsmall';
+    ```
+
+1. Use the [`SHOW CLUSTER REPLICAS`](/sql/show-cluster-replicas/) command to check the status of the new replica:
+
+    ```sql
+    SHOW CLUSTER REPLICAS WHERE cluster = 'ingest_rds_postgres';
+    ```
+    <p></p>
+
+    ```noftm
+           cluster       | replica |  size  | ready
+    ---------------------+---------+--------+-------
+     ingest_rds_postgres | r1      | xsmall | t
+    (1 row)
+    ```
+
+1. Going forward, you can use the [`mz_source_statistics`](/sql/system-catalog/mz_internal/#mz_source_statistics) table to verify that your new replica size is sufficient for Materialize to keep up with the change events coming through the replication stream:
+f
+    ```sql
+    SELECT
+      SUM(updates_staged) AS updates_staged,
+      SUM(updates_committed) AS updates_committed
+    FROM mz_internal.mz_source_statistics
+    WHERE id = '<source_ID>';
+    ```
+
+    <p></p>
+
+    ```nofmt
+     updates_staged | updates_committed
+    ----------------+-------------------
+                  0 |                 0
+    (1 row)
+    ```
+
+    A large gap between `updates_staged` and `updates_committed` is usually an indication that the source has fallen behind and that you might need to scale up your source.
+
+## Next steps
+
+With Materialize ingesting your PostgreSQL data into durable storage, you can start exploring the data, computing real-time results that stay up-to-date as new data arrives, and serving results in various ways.
+
+- Explore
```

Comment:
Just my two cents but this is an overwhelming number of choices; if you had metrics on a page like this, I bet you'd drive more engagement with one or two choice links

Reply:
Stripping it down somewhat.